### PR TITLE
updates take 2

### PR DESCRIPTION
**Pull request by nmoorenz**
I think this is better and github tells me the branches can be automatically merged.

### CODE REVIEW

File: 01_agile-intro.Rmd [hunk 1]
```diff
@@ -5,17 +5,17 @@
...
 Subsequentially these stages are *problem analysis*, *designing the project*, *writing the software*, *testing it* and finally *implementation and maintenance*. 
 [Here](https://www.youtube.com/watch?v=LxEmGNgqYJA) you can find a clear and neutral introducion into the Waterfall approach. 
-What stands out at Waterfall is delivering complete and faultless software. 
+What stands out for Waterfall is delivering complete and faultless software. 
 In order to do so, the centre of gravity of a Waterfall project is the documentation, in which every requirement and aspect of the software is written down meticulously. 
 The underlying conviction is that the software is written faster and is of higher quality when all aspects of it are decided upon upfront.
 
 Projects done with the Waterfall method have a major pitfall, however. 
-They can take very long to be completed. 
+They can take a very long to be completed.
```
Comment (EdwinTh) [inline 01_agile-intro.Rmd]: You mean ` a very long time` here I guess?

File: 05_code_organisation.Rmd [hunk 2]
```diff
@@ -43,4 +43,4 @@ The functions from the package are readily available here.
 
 ## Further Reading
 
-Hadley Wickham wrote a comprehensive book on developing software in R packages. It is freely available online, [here](http://r-pkgs.had.co.nz/). You should read it!
+Hadley Wickham wrote a comprehensive book on developing software in R packages. It is freely available online, [here](http://r-pkgs.had.co.nz/). You should read it! <!--- soon to be updated to https://r-pkgs.org/ --->
```
Comment (EdwinTh) [inline 05_code_organisation.Rmd]: Didn't know that!

File: 06_automation.Rmd [hunk 3]
```diff
@@ -13,41 +13,41 @@ The obvious reason data science products should be automated is that it will sav
...
 This is not only a waste of time, but also a waste of focus.
-Once you have reached a stage that the product is of satisfactory quality and can run without interference for at time, you should be able to move on to a new project without thinking too much about it.
+Once you have reached a stage that the product is of satisfactory quality and can run without interference for a time, you should be able to move on to a new project without thinking too much about it.
 However, if the product is run manually you have to keep spending a part of your attention to the earlier product.
 Continuously switching between projects is strenuous and not satisfying.
 
 Not automating also creates a risk for the continuity of the product.
-It is not healthy to be completely dependent on the knowledge and actions of a single data scientist, because it is not guaranteed he will continue to work at the organisation for the duration of the product.
+It is not healthy to be completely dependent on the knowledge and actions of a single data scientist, because it is not guaranteed they will continue to work at the organisation for the duration of the product.
```
Comment (EdwinTh) [inline 06_automation.Rmd]: Shouldn't `data scientist` be plural in order to refer to it with `they`?

**Comment by EdwinTh** [comment]
Again many thanks for finding all my embarrassing `then` vs `than` errors, really appreciate the time you took to meticulously check the spelling. Just two small things, in de code comments, if they are resolved we are ready to merge.

File: 01_agile-intro.Rmd (hunk 1 above)
Comment (nmoorenz) [reply 01_agile-intro.Rmd]: Yes, a very long time

File: 06_automation.Rmd (hunk 3 above)
Comment (nmoorenz) [reply 06_automation.Rmd]: It's a reasonably recent development that they/them can be singular. It doesn't matter if you keep he

**Pull request merged**